src/vue/elements/boxes.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+
+    <div class="ods-container__box"
+        :class="sizeBox">
+        
+        <div class="ods-container__box-icon">
+            
+            <img :class="sizeIcon"
+                :src="img"
+                alt="OpenDataSoft Documentation"/>
+
+        </div>
+        
+        <div class="ods-container__box-desc">
+            
+            <h2 class="ods-container__box-desc-h2"
+                id="index-title-platform">
+                {{ content.title[lang] }}
+            </h2>
+
+            <p class="ods-container__box-desc-p"
+                id="index-desc-platform"
+                v-if="page === 'home'">
+                {{ content.desc[lang] }}
+            </p>
+            <p class="ods-container__box-desc-p"
+                id="index-desc-platform"
+                v-else>
+                <span v-html=content.desc[lang]></span>
+            </p>
+
+        </div>
+
+    </div>
+
+</template>
+
+<script>
+export default {
+    name: 'boxes',
+    props: ['lang', 'content', 'img', 'page'],
+    computed: {
+        sizeBox: function () {
+            if (this.page === 'home') return 'ods-container__box-index';
+            else return 'ods-container__box-apis';
+        },
+        sizeIcon: function () {
```

Comment:
I think there should be a better solution, at least we could specify the CSS class from the parent component. We can talk about this when you have a moment